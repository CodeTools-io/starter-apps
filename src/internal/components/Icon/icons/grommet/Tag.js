
import React from 'react';
import { Tag } from 'grommet-icons';

export default function TagIcon({...props}) {
  return <Tag {...props}/>
}
  