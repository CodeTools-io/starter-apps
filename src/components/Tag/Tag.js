import React from 'react';
import { Box, Text } from 'grommet';

export default function Tag({ color, label, ...props }) {
  return (
    <Box
      className="Tag"
      background={color}
      pad={{
        horizontal: 'small',
        vertical: 'xsmall',
      }}
      {...props}
    >
      <Text size="small">{label}</Text>
    </Box>
  );
}
