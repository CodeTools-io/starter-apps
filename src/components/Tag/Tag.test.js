import React from 'react'
import Tag from './Tag'

describe('Tag', () => {
  test('can render', () => {
    const { container } = renderComponent(
      <Tag>
        <p>Content for the Tag</p>
      </Tag>
    )

    expect(container).toMatchSnapshot()
  })
})

