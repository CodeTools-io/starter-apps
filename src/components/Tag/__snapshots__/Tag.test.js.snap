// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag can render 1`] = `
<div>
  <div
    class="StyledBox-sc-13pk1d4-0 dAnYqx Tag"
  >
    <span
      class="StyledText-sc-1sadyjn-0 hncrgS"
    />
  </div>
</div>
`;
